### ISSUE TITLE
Incorrect grid width variable in docs ($theme-site-max-width vs $theme-grid-container-max-width)

### ISSUE THREAD

**Issue by jonathanfann** [uswds/uswds-site#1038]
I'm writing some documentation on the grid to help our devs and have questions about this line on the docs (https://designsystem.digital.gov/utilities/layout-grid/)
`Set the default max width with $theme-site-max-width in uswds-theme-spacing.scss.`

However, when I search the project for that variable, I don't see any references to it (outside of README / declarations).  I do see references to `$theme-grid-container-max-width`.  The site should reflect this variable in that sentence now.

Long story short,

`Set the default max width with $theme-site-max-width in uswds-theme-spacing.scss.` should read `Set the default max width with $theme-grid-container-max-width in uswds-theme-spacing.scss.` (also maybe remove the "old" $theme-site-max-width since it isn't used anywhere?  or just set it = $theme-grid-container-max-width?)
